src/analyzer/plugins/analyzerkeyfinder.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+#include <keyfinder/keyfinder.h>
+
+#include <QObject>
+
+#include "analyzer/plugins/analyzerplugin.h"
+#include "util/types.h"
+
+namespace mixxx {
+
+class AnalyzerKeyFinder : public AnalyzerKeyPlugin {
+  public:
+    static AnalyzerPluginInfo pluginInfo() {
+        return AnalyzerPluginInfo(
+                "keyfinder:2",
```

Comment:
QStringLiteral()? I suggest to move this implementation into the .cpp file and declare the strings as constants in an anonymous namespace.

Reply:
This is static. It must be in the class definition.